src/Type/Components/MapTypes.ts

```diff
@@ -0,0 +1,26 @@
+export interface MarkerData {
+    position: [number, number]
+    popupContent: string
+}
+
+export interface CustomSelectPositionMapProps {
+    onPositionChange: (lat: number, lng: number) => void
+    center?: [number, number]
+    zoomLevel?: number
+}
+
+export interface DisplayPositionProps {
+    map: any
+    onPositionChange: (lat: number, lng: number) => void
```

Comment:
this

Reply:
this also am doing as per map conditions

Reply:
expecting lat , long in props